index.html

```diff
@@ -43,6 +43,139 @@ <h3 class="title dtitle">I'm a software developer.</h3>
                 </div> 
             </section>
         </div>
+
+        <section class="recent-works">
+            <div class="recent-works-header">
+                <h2 class="title">My Recent works</h2>
+                <hr class="recent-works-line">
+            </div>
+
+            <div class="cards">
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <h3 class="card-title">Multi-Post Stories Gain + Glory</h3>
+                        <ul class="card-tools-list">
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project btn">See project</button>
+                    </div>
+                </div>
```

Comment:
- [ ] **[OPTIONAL]** I noticed that the height of your work cards is set to `90vh`. This could cause some issues with smaller devices since the cards would take up a lot of space. As a suggestion, you could consider removing the `height` property and instead use some `padding-bottom` for your text. This way, your cards will be more responsive and will look great on all devices. Keep up the great work! 👌

### Your design (375x752):
![image](https://user-images.githubusercontent.com/107952724/235332064-3fadf780-a775-42a2-a46a-4479bf9309cd.png)

### Figma design:
![image](https://user-images.githubusercontent.com/107952724/235332044-5b7de161-d484-44a5-aa07-120267f213e1.png)

  _Just a friendly reminder that adhering as closely as possible to the Figma template is a requirement for this project. While you've done a great job so far, it's important to keep the template in mind as you continue with your design._
  ![image](https://user-images.githubusercontent.com/107952724/235323387-c50f856e-6774-45ca-ae81-7304c8b62709.png)